src/mmrelay/cli_utils.py

```
@@ -30,7 +30,7 @@
 try:
     import certifi
 except ImportError:
-    certifi = None
+    certifi: Module | None = None
```

Ruff (0.14.10)

Undefined name `Module`

(F821)